cssselect2/extensions.py

```diff
@@ -7,6 +7,8 @@
 def _match(selector):
     "Callback for match pseudoClass."
     regex = serialize(selector.arguments)
+    if regex.startswith('"') and regex.endswith('"'):
+      regex = regex[1:-1]
```

Comment:
To cover both, something like this:
if regex[1] in ['"','"'] and regex[1] == regex[-1]